schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/layout/utility-fn.xsl

```diff
@@ -122,9 +122,14 @@
       <xsl:when test="matches($string, concat('.*', $translationSeparator, '.*'))">
         <xsl:for-each select="tokenize($string, $translationSeparator)">
           <xsl:variable name="descLang"
-                        select="substring-before(., $valueSeparator)"/>
+                        select="if ($valueSeparator != '')
+                                then substring-before(., $valueSeparator)
+                                else substring(., 1, 2)"/>
           <xsl:variable name="descValue"
-                        select="substring-after(., $valueSeparator)"/>
+                        select="if ($valueSeparator != '')
+                                then substring-after(., $valueSeparator)
+                                else substring(., 4)"/>
```

Comment:
What means the value 4? If not obvious, can you add a code comment to explain it?

Reply:
Added description https://github.com/geonetwork/core-geonetwork/pull/6588/commits/80fbd5f7da3ea517a32b9d711a41612693f61e14

it is a particular case for the onlinesrc-add.xsl which can add URL containing the separator '#'